<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card</class>
 <widget class="QWidget" name="card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_LIFE">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生命</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_COST">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>费用</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ACK">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>攻击</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
